bridge/opentracing/bridge.go

```diff
@@ -154,7 +154,15 @@ func (s *bridgeSpan) SetOperationName(operationName string) ot.Span {
 func (s *bridgeSpan) SetTag(key string, value interface{}) ot.Span {
 	switch key {
 	case string(otext.SpanKind):
-		// TODO: Should we ignore it?
+		type readSpanKindSpan interface {
+			SpanKind() trace.SpanKind
+		}
+		// only add span kind tag when otel span kind is unset or bridge default (internal)
+		if span, ok := s.otelSpan.(readSpanKindSpan); ok {
+			if span.SpanKind() == trace.SpanKindUnspecified || span.SpanKind() == trace.SpanKindInternal {
+				s.otelSpan.SetAttributes(otTagToOTelAttr(key, value))
```

Comment:
This will cause the `span.kind` to be duplicated at the exporter level. This will set it in the attributes, but the `recordingSpan.spanKind` will also be set to the default `internal` - both of these seem to get sent via the exporter. I've tested this with `go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc` sending to Jaeger collector.